### PR TITLE
Fixes Rendered Gateway Example

### PR DESCRIPTION
**Pull request by danehans**
Updates the rendered gateway manifest script to include manifests defined in the examples/gateway directory.

Signed-off-by: Daneyon Hansen <daneyonhansen@gmail.com>

### CODE REVIEW

File: examples/render/contour-gateway.yaml [hunk 1]
```diff
@@ -3167,3 +3167,3340 @@ spec:
           secret:
             secretName: envoycert
       restartPolicy: Always
+
```
Comment (skriss) [inline examples/render/contour-gateway.yaml:3170]: I think we need a `---` separator here; the component Contour files all include them but Gateway CRDs don't have one at the top

**Comment by danehans** [comment]
@skriss thanks for the review. Commit `8f9f787` resolves your feedback.

**Review by skriss — APPROVED** [review]
:+1: LGTM. We could cherry-pick this to the release-1.16 branch if you want too, to fix the manifests there. Just create another PR with the same commit against `release-1.16`.

**Comment by danehans** [comment]
> We could cherry-pick this to the release-1.16 branch if you want too, to fix the manifests there.

@skriss https://github.com/projectcontour/contour/pull/3664 didn't land in v1.16.

**Comment by skriss** [comment]
Ah, ok, I thought it had. nvm!

**Review by youngnick — APPROVED** [review]

**Pull request merged**